PetGame/Controllers/UserController.cs

```diff
@@ -52,19 +52,12 @@ public IActionResult GetWhoAmI()
         //    return Ok();
         //}
 
-        // creates a new pet for the current user
-        [HttpPost("Pet")]
-        public IActionResult PostPet([FromForm, FromBody] CreatePetModel model)
+        [HttpGet("{id}/status")]
+        public IActionResult Get(ulong id)
```

Comment:
Why is the `id` in here? You really should only get the status for the _current_ user. Otherwise, you are just letting anyone get the access of other user's pets.

Reply:
So, you shouldn't have to specify the id in the query string at all.